server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/CreateMissingParentsTask.java

```diff
@@ -119,13 +138,45 @@ public CreateMissingParentsTask(MailboxManager mailboxManager) {
 
     @Override
     public Result run() {
```

Comment:
No test to verify if the task does actually create missing parents mailboxes?

Reply:
the fact that the "created" array has some mailboxIds in it is not convincing enough?